scripts/qa_crowbarsetup.sh

```diff
@@ -2908,6 +2908,12 @@ function onneutron_wait_for_neutron()
     fi
 }
 
+function onnovacontroller_wait_for_nova()
+{
+    get_novacontroller
+    wait_for 300 3 "ssh $novacontroller 'service openstack-nova-api status'" "nova-api service running state"
```

Comment:
I guess you also want to wait for nova-manage service list being sucessful, see testsetup function. perhaps we can use that same code here as well (e.g. putting it in a common function)